contracts/src/starknet/rewaave/tokens/ETHstaticAToken.cairo

```diff
@@ -140,9 +147,16 @@ func burn{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
 end
 
 func claim_rewards{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
-        user : felt, recipient : felt) -> (claimed : Uint256):
-    Ownable_only_owner()
-    return claimable_claim_rewards(user, recipient)
+        user : felt, recipient : felt) -> (success : felt):
+    let (caller) = get_caller_address()
+
+    with_attr error_message("user should be {caller}"):
+        assert caller = user
```

Comment:
If the `user` should be the caller just use `caller` instead of passing an argument